app/src/main/java/com/realityexpander/tasky/TaskyApplication.kt

```diff
@@ -31,7 +31,9 @@ class TaskyApplication: Application(), Configuration.Provider {
         AndroidLogcatLogger.installOnDebuggableApp(this, minPriority = LogPriority.VERBOSE)
 
         logcat { "TaskyApplication.onCreate(): starting Tasky workers" }
-        startSyncAgendaWorker(applicationContext)
-        startRefreshAgendaWeekWorker(applicationContext)
+//        startSyncAgendaWorker(applicationContext)
+//        startRefreshAgendaWeekWorker(applicationContext)
+        SyncAgendaWorker.startWorker(applicationContext)
+        RefreshAgendaWeekWorker.startWorker(applicationContext)
```

Comment:
What if the user isn't logged in? 

I recommend starting the workers on the AgendaViewModel (Or however it was called), and cancelling them on logout